packages/common/src/templates/icons.ts

```diff
@@ -118,6 +120,8 @@ export default function getIcon(theme: TemplateType): ReturnedIcon {
       return MDXDeckIcon;
     case quasar.name:
       return QuasarIcon;
+    case unibit.name:
+      return StackbitIcon;
```

Comment:
Update Icon Reference

```suggestion
      return UnibitIcon;
```